.github/workflows/run-integration-tests.yaml

```diff
@@ -0,0 +1,131 @@
+name: Run E2E integration Tests
+
+# This workflow is meant to be called from other repositories' workflows to run integration checks autonomously.
+# While this workflow makes use of secrets, this workflow DOES NOT pull secrets itself and expects upstream callers to
+# provide the secrets via the github secrets functionality.
+#
+# The caller repository must have secrets set up for (Workload Identity Federation is not available for this suite atm)
+#        LYLE_CLIENT_EMAIL
+#        LYLE_PRIVATE_KEY
+#        FIRECLOUD_ACCOUNT_CLIENT_EMAIL
+#        FIRECLOUD_ACCOUNT_PRIVATE_KEY
+# Instruction for getting these secrets securely from vault to github secrets can be found here:
+# https://docs.google.com/document/d/1JbjV4xjAlSOuZY-2bInatl4av3M-y_LmHQkLYyISYns/edit?usp=sharing
+#
+# With that configured, here's how you can call this workflow from whatever workflow currently publishes the app:
+# ```yaml
+# jobs:
+#
+#
+#   This example runs just the register-user.js tests against terra-dev
+#   run-terra-ui-integration-tests:
+#     uses: databiosphere/terra-ui/.github/workflows/run-integration-tests.yaml@main
+#     permissions:
+#       contents: 'read'
+#      with:
+#        test_url: 'https://bvdp-saturn-dev.appspot.com/'
+#        environment: 'dev'
+#        tests_to_run: 'tests/register-user.js'
+#      secrets:
+#        LYLE_CLIENT_EMAIL: ${{ secrets.LYLE_CLIENT_EMAIL }}
+#        LYLE_PRIVATE_KEY: ${{ secrets.LYLE_PRIVATE_KEY }}
+#        FIRECLOUD_ACCOUNT_CLIENT_EMAIL: ${{ secrets.FIRECLOUD_ACCOUNT_CLIENT_EMAIL }}
+#        FIRECLOUD_ACCOUNT_PRIVATE_KEY: ${{ secrets.FIRECLOUD_ACCOUNT_PRIVATE_KEY }}
+# ```
+
+on:
+  workflow_call:
+    inputs:
+
+      ##
+      ## Configurations:
+      ##
+
+      test_url:
+        required: true
+        type: string
+        description: "URL of terraui instance to test against, required"
```

Comment:
Is the intent to usually run these tests against dev, alpha, staging? Or BEEs? If the former, it would be nice to derive `test_url` from `environment` instead of having to specify two parameters that must match.

Reply:
The intent here is to have this work all 4 scenarios. I would like to give teams the options to run against BEEs in PR environments to cut down on post-PR errors.